external-crates/move/crates/move-vm-runtime/src/execution/values/values_impl.rs

```diff
@@ -183,17 +177,12 @@ struct VectorMatchRefMut<'v>(VectorMatch<&'v mut Vec<MemBox<Value>>, &'v mut Pri
 #[derive(Debug)]
 pub enum GlobalValueImpl {
     /// No resource resides in this slot or in storage.
-    None,
-    /// A resource has been published to this slot and it did not previously exist in storage.
-    Fresh { container: MemBox<Value> },
-    /// A resource resides in this slot and also in storage. The status flag indicates whether
-    /// it has potentially been altered.
-    Cached {
-        fingerprint: GlobalFingerprint,
-        container: MemBox<Value>,
-    },
-    /// A resource used to exist in storage but has been deleted by the current transaction.
-    Deleted,
+    Empty,
+    /// A resource exists at this slot. No information about the state of storage (e.g., if it is
+    /// new or was an existing value that was loaded) as it relates to this value is implied.
+    /// Determining if this is a new or mutated global value a the end of execution is the
+    /// responsibility of the runtime.
```

Comment:
"The runtime" is ambiguous here. Please explain a bit what is actually going to happen here.